### PR TITLE
docs(web): improve descriptions of the api

### PR DESCRIPTION
**Pull request by m8vago**
Remove `deployments/:deploymentId/events` as it was the duplication of `/log`
Improve api docs.

### CODE REVIEW

File: web/crux/src/app/image/image.http.controller.ts [hunk 1]
```diff
@@ -68,10 +68,10 @@ export default class ImageHttpController {
   @HttpCode(200)
   @ApiOperation({
     description:
-      "Fetch details of an images within a version. `ProductId` refers to the product's ID, `versionId` refers to the version's ID, `imageId` refers to the image's ID. All are required variables.</br></br>Details come in an array, including `name`, `id`, `tag`, `order`, and config details of the image.",
+      "Fetch details of an images within a version. `productId` refers to the product's ID, `versionId` refers to the version's ID, `imageId` refers to the image's ID. All are required parameters.</br></br>An image consists `name`, `id`, `tag`, `order`, and config details of the image.",
```
Comment (gerimate) [inline web/crux/src/app/image/image.http.controller.ts:71]: Should be (fixed part bolded):
- Fetch details of an **image** within a version.
- Image details consists of `name`, `id`, `tag`, `order`, and config details of the image.

File: web/crux/src/app/team/team.http.controller.ts [hunk 2]
```diff
@@ -127,14 +127,14 @@ export default class TeamHttpController {
   @CreatedWithLocation()
   @ApiOperation({
     description:
-      "Request must include `teamId`, email and `firstName`. Admin access required for successful request.</br></br>Response should include new user's `name`, `id`, `role`, `status`, `email`, and `lastLogin`.",
-    summary: 'Add new user to a team.',
+      "Request must include `teamId`, email and `firstName`. Admin access required for successful request.</br></br>Response should include new user's `name`, `id`, `role`, `status`, `email`, and `lastLogin`. Admin access required for successful request.",
```
[the remark this answers is not in the document]
Comment (m8vago) [reply web/crux/src/app/team/team.http.controller.ts:130]: @gerimate ?

[the remark this answers is not in the document]
Comment (gerimate) [reply web/crux/src/app/team/team.http.controller.ts:130]: Both good, but I prefer BigG's suggestion

**Review by gerimate — APPROVED** [review]
LGTM

**Pull request merged**